crates/bevy_gltf/Cargo.toml

```
@@ -29,4 +29,4 @@ thiserror = "1.0"
 anyhow = "1.0.4"
 base64 = "0.13.0"
 percent-encoding = "2.1"
-wgpu = "0.11.0"
+wgpu = { version = "0.12.0" }
```

```suggestion
wgpu = "0.12.0"
```
looks a bit nicer and is more consistent.

---

fixed